# Define the targeted pose
geometry_msgs/Pose pose_target

---
# Define the result
bool result
moveit_msgs/MoveItErrorCodes error_code

---
# Define a feedback message
string state